DiscordChatExporter.Domain/Discord/Models/ChannelCategory.cs

```diff
@@ -44,5 +44,10 @@ public static ChannelCategory Parse(JsonElement json, int? position = null)
                 position.Value
             );
         }
+
+        public static ChannelCategory Empty()
+        {
+            return new (Snowflake.Zero, "Missing", 0);
+        }
```

Comment:
Let's make it into a property:

```suggestion
        public static ChannelCategory Empty { get; } = new(Snowflake.Zero, "Missing", 0);
```

Reply:
Nice, I have not touched C# for a few years now and was not sure what syntactic sugar was available. Thanks!